Makefile

```diff
@@ -230,8 +216,7 @@ KOMPILE_OPTS      :=
 LLVM_KOMPILE_OPTS :=
 
 build: build-llvm build-haskell build-java build-web3 build-node
-build-all: build build-ocaml
-build-ocaml:   $(ocaml_kompiled)
+build-all: build
```

Comment:
Just remove the `build-all` target and have `Jenkinsfile` call `build` directly. Make sure to remove it from `.PHONY` too.